client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -55,15 +55,14 @@ class AuthorMappingPane extends PureComponent {
 						destinationSiteTitle: targetTitle,
 					},
 					components: {
-						b: <strong />,
 						em: <em />,
 					},
 				}
 			);
 		} else if ( numTargetUsers === 1 && numSourceUsers > 1 ) {
 			return this.props.translate(
 				'There are multiple authors on your %(sourceType)s site. ' +
-					"Because you're the only author on {{b}}%(destinationSiteTitle)s{{/b}}, " +
+					"Because you're the only author on your new site, " +
```

Comment:
As elsewhere, can we use "your original FOO site. Because you're the only author on this site (**example.wordpress.com**), "?
```suggestion
				'There are multiple authors on your original %(sourceType)s site. ' +
					"Because you're the only author on this site (**example.wordpress.com**), " +
```